infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -488,10 +615,10 @@ private LogData getLogData(LogEntry entry) {
      * @return metadata field of null if it was partially written.
      * @throws IOException IO exception
      */
-    private Metadata parseMetadata(FileChannel fileChannel, String segmentFile) throws IOException {
+    static Metadata parseMetadata(FileChannel fileChannel, String segmentFile) throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651450)